setup/triggers/github/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: single
+title:  "Configuring GitHub Webhooks"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker can be configured to listen to changes to a repository in
+[GitHub](https://github.com). These steps will take you through how to
+configure webhook push events to be sent to Spinnaker from a single repository.
```

Comment:
"These steps show how to configure webhook push events to send to Spinnaker from a single repository."